pkg/juicefs/juicefs.go

```diff
@@ -607,6 +609,32 @@ func (j *juicefs) JfsUnmount(ctx context.Context, volumeId, mountPath string) er
 	return nil
 }
 
+func (j *juicefs) CreateTarget(ctx context.Context, target string) error {
+	var corruptedMnt bool
+	var exists bool
+
+	for {
+		err := util.DoWithTimeout(ctx, defaultCheckTimeout, func() (err error) {
+			exists, err = k8sMount.PathExists(target)
+			return
+		})
+		if err == nil {
+			if !exists {
```

Comment:
If `err == nil`, we can straightly call `MkdirAll`, which will deal with existed path.